Tests/Pcap++Test/Tests/PfRingTests.cpp

```diff
@@ -328,7 +328,7 @@ PTF_TEST_CASE(TestPfRingDeviceMultiThread)
 	aggrStats.drop = 0;
 
 	pcpp::PfRingDevice::PfRingStats stats;
-	for (int i = 0; i < totalnumOfCores; i++)
+	for (i = 0; i < totalnumOfCores; i++)
```

Comment:
I think it's better to keep the i local to the for loop and change the while loop above with also a for loop